docs-library/10x-Governance.md

```diff
@@ -0,0 +1,181 @@
+# How 10x Projects are Selected at every Phase
+## Why do we use a phased approach?
+Risk is an inherent element of new product development. The process of developing a new product or service can be seen as an act of continually de-risking a product or service by systematically identifying and addressing the risks that could keep it from succeeding.
+
+For example, every new product or service starts with an assumption about who the target customer is and what their desires are. This assumption, while a necessary starting point, is rarely accurate enough to generate a successful product that works for its intended market. Initial user, market, and product research can help the product team adapt these initial assumptions about the target customer to reflect the reality of who the customer actually is. Doing this research de-risks future product development because an accurate understanding of the customer base removes the risk of making incorrect product development choices based on incorrect assumptions about the customer.
+
+The goal of the 10x “incremental investment” process is to **maximize the likelihood that every project team de-risks the product or service to the greatest extent possible**, by systematically gating the financial commitment to the project based on proof that the project has addressed the clearest sources of risk at every stage. The level of risk is at its highest in early-stage projects, where a high degree of uncertainty exists about whether the product will work or not, and each subsequent phase should be able to demonstrate materially less risk in order to continue to fund the project. 
+
+As a project advances through the phases, the project team must show that they have identified and addressed the types of project risks typical to projects at each phase. Alongside the project team, the 10x team is an active participant in helping teams identify risks associated with their project at each stage, and in ensuring that teams are able to successfully identify how these risks can best be addressed.
+
+##What criteria are used to fund 10x projects?
```

Comment:
be sure to include a space after the `##` or GitHub won't parse the markdown correctly 😛